src/device/gl_device/lib.rs

```diff
@@ -575,19 +565,36 @@ impl Device for GlDevice {
 
     fn reset_state(&mut self) {
         let data = ::draw::DataBuffer::new();
-        for com in RESET_CB.iter() {
-            self.process(com, &data);
-        }
-    }
-
-    fn submit(&mut self, (cb, db): (&GlCommandBuffer, &::draw::DataBuffer)) {
+        self.process(&Command::BindProgram(0), &data);
```

Comment:
If `Command` is not longer `Send` does that not screw up our ability to send to a device located on another thread?